### PR TITLE
CO-1167 FIX - Remove redondant workflow activity "act_cancelled"

### PR DESCRIPTION
**Pull request by eicher31**

### CODE REVIEW

**Review by ecino — APPROVED** [review]

**Comment by ecino** [comment]
In fact there is a problem in line https://github.com/eicher31/compassion-modules/blob/5e351a1a78819ecc7c85602d58a6367c2219b4eb/sponsorship_tracking/workflow/sds_workflow.xml#L102 and below where you must change reference to `act_cancelled` by `contract_compassion.act_cancelled`

File: sbc_compassion/models/import_letter_line.py [hunk 1]
```diff
@@ -25,9 +25,9 @@ class ImportLetterLine(models.Model):
     ##########################################################################
 
     sponsorship_id = fields.Many2one('recurring.contract', 'Sponsorship',
-                                     compute='_set_sponsorship_id')
+                                     compute='_compute_set_sponsorship_id')
```
Comment (ecino) [inline sbc_compassion/models/import_letter_line.py]: compute_sponsorship would be better

File: sbc_compassion/models/import_letter_line.py [hunk 2]
```diff
@@ -25,9 +25,9 @@ class ImportLetterLine(models.Model):
...
     partner_id = fields.Many2one('res.partner', 'Partner')
-    name = fields.Char(compute='_set_name')
+    name = fields.Char(compute='_compute_set_name')
```
Comment (ecino) [inline sbc_compassion/models/import_letter_line.py]: _compute_name would be better

File: sbc_compassion/models/import_letters_history.py [hunk 3]
```diff
@@ -55,7 +55,7 @@ class ImportLettersHistory(models.Model):
         track_visibility='onchange')
     import_completed = fields.Boolean()
     nber_letters = fields.Integer(
-        "Number of files", readonly=True, compute="_count_nber_letters")
+        "Number of files", readonly=True, compute="_compute_count_nber_letters")
```
Comment (ecino) [inline sbc_compassion/models/import_letters_history.py]: _compute_nber_letters would be better

File: sbc_compassion/wizards/import_review.py [hunk 4]
```diff
@@ -23,12 +23,12 @@ class ImportReview(models.TransientModel):
     ##########################################################################
     #                                 FIELDS                                 #
     ##########################################################################
-    progress = fields.Float(compute='_get_current_line', store=True)
+    progress = fields.Float(compute='_compute_get_current_line', store=True)
```
Comment (ecino) [inline sbc_compassion/wizards/import_review.py]: _compute_current_line

**Review by ecino — APPROVED** [review]
Small propositions not mandatory...

File: sbc_compassion/wizards/import_letters_test.py [hunk 5]
```diff
@@ -61,7 +61,7 @@ def _analyze_attachment(self, file_, filename):
             })
             letters_line.letter_image = self.env[
                 'ir.attachment'].create(document_vals[i])
-            letters_line.check_status()
+            letters_line._compute_check_status()
```
Comment (eicher31) [inline sbc_compassion/wizards/import_letters_test.py:64]: I'm not sure if its the right way that access to a private methode

File: sbc_compassion/wizards/import_review.py (hunk 4 above)
Comment (eicher31) [reply sbc_compassion/wizards/import_review.py]: I agree

File: sbc_compassion/wizards/import_letters_test.py (hunk 5 above)
Comment (ecino) [reply sbc_compassion/wizards/import_letters_test.py:64]: It's not a problem from Odoo code to access private methods of Odoo objects. Private methods won't be accessible remotely with XML-RPC.

**Comment by ecino** [comment]
:+1:

**Pull request merged**